srml/support/src/storage/generator.rs

```diff
@@ -553,6 +581,118 @@ macro_rules! decl_storage {
+		__decl_genesis_config_items!($traitinstance $traittype [ $($t)* $getfn : $ty = $default ; ] $($res)* );
+	};
+
+	// maps
+	($traitinstance:ident $traittype:ident [$($t:tt)*] $(#[$doc:meta])* $name:ident : default map [$kty:ty => $ty:ty]; $($res:tt)*) => {
+		__decl_genesis_config_items!($traitinstance $traittype [ $($t)* ] $($res)* );
+	};
+	($traitinstance:ident $traittype:ident [$($t:tt)*] $(#[$doc:meta])* pub $name:ident : default map [$kty:ty => $ty:ty]; $($res:tt)*) => {
+		__decl_genesis_config_items!($traitinstance $traittype [ $($t)* ] $($res)* );
+	};
+	($traitinstance:ident $traittype:ident [$($t:tt)*] $(#[$doc:meta])* $name:ident : required map [$kty:ty => $ty:ty]; $($res:tt)*) => {
+		__decl_genesis_config_items!($traitinstance $traittype [ $($t)* ] $($res)* );
+	};
+	($traitinstance:ident $traittype:ident [$($t:tt)*] $(#[$doc:meta])* pub $name:ident : required map [$kty:ty => $ty:ty]; $($res:tt)*) => {
+		__decl_genesis_config_items!($traitinstance $traittype [ $($t)* ] $($res)* );
+	};
+	($traitinstance:ident $traittype:ident [$($t:tt)*] $(#[$doc:meta])* $name:ident : map [$kty:ty => $ty:ty]; $($res:tt)*) => {
+		__decl_genesis_config_items!($traitinstance $traittype [ $($t)* ] $($res)* );
+	};
+	($traitinstance:ident $traittype:ident [$($t:tt)*] $(#[$doc:meta])* pub $name:ident : map [$kty:ty => $ty:ty]; $($res:tt)*) => {
+		__decl_genesis_config_items!($traitinstance $traittype [ $($t)* ] $($res)* );
+	};
+
+	($traitinstance:ident $traittype:ident [$($t:tt)*] $(#[$doc:meta])* $name:ident get($getfn:ident) : default map [$kty:ty => $ty:ty]; $($res:tt)*) => {
+		__decl_genesis_config_items!($traitinstance $traittype [ $($t)* ] $($res)* );
+	};
+	($traitinstance:ident $traittype:ident [$($t:tt)*] $(#[$doc:meta])* pub $name:ident get($getfn:ident) : default map [$kty:ty => $ty:ty]; $($res:tt)*) => {
+		__decl_genesis_config_items!($traitinstance $traittype [ $($t)* ] $($res)* );
+	};
+	($traitinstance:ident $traittype:ident [$($t:tt)*] $(#[$doc:meta])* $name:ident get($getfn:ident) : required map [$kty:ty => $ty:ty]; $($res:tt)*) => {
+		__decl_genesis_config_items!($traitinstance $traittype [ $($t)* ] $($res)* );
+	};
+	($traitinstance:ident $traittype:ident [$($t:tt)*] $(#[$doc:meta])* pub $name:ident get($getfn:ident) : required map [$kty:ty => $ty:ty]; $($res:tt)*) => {
+		__decl_genesis_config_items!($traitinstance $traittype [ $($t)* ] $($res)* );
+	};
+	($traitinstance:ident $traittype:ident [$($t:tt)*] $(#[$doc:meta])* $name:ident get($getfn:ident) : map [$kty:ty => $ty:ty]; $($res:tt)*) => {
+		__decl_genesis_config_items!($traitinstance $traittype [ $($t)* ] $($res)* );
+	};
+	($traitinstance:ident $traittype:ident [$($t:tt)*] $(#[$doc:meta])* pub $name:ident get($getfn:ident) : map [$kty:ty => $ty:ty]; $($res:tt)*) => {
+		__decl_genesis_config_items!($traitinstance $traittype [ $($t)* ] $($res)* );
+	};
+
+	// exit
+	($traitinstance:ident $traittype:ident []) => {
+		__generate_genesis_config!(WITHOUT_GENESIS);
```

Comment:
This call it not required, as you not do anything in the `WITHOUT_GENESIS` arm.
I would completely remove the `WITHOUT_GENESIS` arm in the `__generate_gensis_config!` macro.